crates/libcgroups/src/common.rs

```diff
@@ -335,10 +338,16 @@ pub struct CgroupConfig {
     pub container_name: String,
 }
 
-pub fn create_cgroup_manager(
+pub fn create_cgroup_manager_with_root(
```

Comment:
Please comment on the behavior of passing None

Reply:
done